packages/next/src/server/app-render/app-render.tsx

```diff
@@ -635,7 +637,26 @@ function ReactServerEntrypoint<T>({
     )
   )
 
-  return <AppRouter initialRSCPayload={response} />
+  const initialState = createInitialRouterState({
+    buildId: response.b,
+    initialFlightData: response.f,
+    initialCanonicalUrl: response.c,
+    // location and initialParallelRoutes are not initialized in the SSR render
+    // they are set to an empty map and window.location, respectively during hydration
+    initialParallelRoutes: null!,
+    location: null,
+    couldBeIntercepted: response.i,
+  })
+
+  const actionQueue = createMutableActionQueue(initialState)
```

Comment:
server renders don't have updates so the lifting isn't really important to do here